src/main/java/com/onarandombox/MultiverseCore/display/filters/ContentFilter.java

```diff
@@ -0,0 +1,21 @@
+package com.onarandombox.MultiverseCore.display.filters;
+
+/**
+ * Filter display content for it show only certain string.
+ */
+public interface ContentFilter {
+    /**
+     * Checks if a particular string should be displayed.
+     *
+     * @param value String to check on.
+     * @return True if should be display, false otherwise.
+     */
+    boolean checkMatch(String value);
+
+    /**
+     * Gets whether content needs to be filtered by this filter.
+     *
+     * @return True if content should be filtered, false otherwise.
+     */
+    boolean needToFilter();
+}
```

Comment:
First glance, this seems like it could easily be a single method, functional interface. If the need to filter is false, then the checkMatch should just return true. In that case, you could also just use the built in [Predicate interface](https://docs.oracle.com/javase/8/docs/api/java/util/function/Predicate.html).

Reply:
Yea, then reason I d have the second method, is so that there is no need to loop through the content to filter is there nothing to filter. And when displaying to user, it output whether or not filter was applied to the content.

Reply:
Example use cases:
```java
// Only send output of filter string used if avaliable.
if (filter.needToFilter()) {
    sender.sendMessage(String.format("%s[Filter '%s']", ChatColor.GRAY, filter));
}
```

```java
// Only loop to filter if needed, to save performance
if (filter.needToFilter()) {
    return content.stream().filter(filter::checkMatch).collect(Collectors.toList());
}
return content;
```